spec/spec_helper.rb

```diff
@@ -8,6 +8,8 @@ def horizontal_rule(width = 5)
 
 require 'puppetlabs_spec_helper/module_spec_helper'
 
+FUTURE_PARSER = ENV['FUTURE_PARSER'] == 'true' || ENV['FUTURE_PARSER'] == 'yes'
+
```

Comment:
@glennsarti I would not say it is true, as there may be differences. The thing is, we don't need to test that scenario at all as we are not concerned with Future parser testing for the most part. Plus we have 4.x we can test against, which should provide a pretty good gauge of future parser without all of the spec headaches.